Orange/widgets/unsupervised/owmds.py

```diff
@@ -29,6 +32,50 @@ def stress(X, distD):
     return delta_sq.sum(axis=0) / 2
 
 
+class Result(namespace):
+    embedding = None  # type: np.ndarray
+
+
+def run_mds(matrix: DistMatrix, max_iter: int, step_size: int, init_type: int,
+            embedding: np.ndarray, state: TaskState):
+    res = Result(embedding=embedding)
+
+    iterations_done = 0
+    init = embedding
+    state.set_status("Running...")
+    oldstress = np.finfo(np.float).max
+
+    while True:
+        step_iter = min(max_iter - iterations_done, step_size)
+        mds = MDS(
+            dissimilarity="precomputed", n_components=2,
+            n_init=1, max_iter=step_iter,
+            init_type=init_type, init_data=init
+        )
+
+        with warnings.catch_warnings():
+            warnings.filterwarnings(
+                "ignore", ".*double_scalars.*", RuntimeWarning)
```

Comment:
My fault. The consequence, I suppose, could be that the filter is not disabled inside the context, or that it remains disabled after exiting the context.

I concur it should be removed, though it would be nice to avoid this warning. If it is not too complicated, please investigate why it's triggered (in tests, I suppose).

Reply:
The warning appears when fed dataset where all features are constant.

Reply:
Can you prevent it? In tests by not giving such data, in widget by checking it and showing a user warning? It doesn't have to be in this PR, but could you just write a short issue in this case?